### PR TITLE
Add mbe-nodes and angular-ui-layout

### PR DESCRIPTION
**Pull request by nicklasb**
This adds the frontend for MBE and the angular-ui-layout package that it depends upon.

### CODE REVIEW

File: package-overrides/github/angular-ui/ui-layout@1.1.0.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "dependencies": {
+    "angular": "^1"
+  },
+  "registry": "jspm",
+  "main":
+    ["./src/ui-layout.js"],
```
Comment (guybedford) [inline package-overrides/github/angular-ui/ui-layout@1.1.0.json]: jspm doesn't stricly support a main array, may be best to make this a plain string.

File: package-overrides/github/angular-ui/ui-layout@1.1.0.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "shim": {
+    "ui-utils": {
+      "deps": "angular"
+    }
+  },
+  "format":"amd"
```
Comment (guybedford) [inline package-overrides/github/angular-ui/ui-layout@1.1.0.json]: Is it global or AMD here?

**Comment by guybedford** [comment]
Looks great, thanks... just let me know about the minor points above.

**Comment by nicklasb** [comment]
1. I'll change that into a string.
2. Oh. That was certainly strange of me to miss, I'll have to blame that on the sun spots. I am even not sure either is needed anymore, I'll get back to you on that.

**Comment by nicklasb** [comment]
The format doesn't seem to be needed with the new version.

**Comment by guybedford** [comment]
:+1:

**Pull request merged**